pages/changelog.mdx

```diff
@@ -12,6 +12,21 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Upcoming
 
+### 2024-10-30
+
+**CHANGED**
+
+##### Description:
+
+- **Multiple Redirect URLs:** You can now configure multiple redirect URLs for your exchange sessions, providing more flexibility in your integration.
+- **Required Redirect URL:** A redirect URL is now required when [creating an exchange session](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) to ensure proper validation and security.
+
+##### Changes:
+
+- The [create an exchange session API endpoint](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) now requires a redirect url parameter, which is a single URL that is already configured with Dwolla.
```

Comment:
Recommendations on /balance and /connect links ? 🤔

Reply:
Maybe something like this?

```suggestion
- The endpoint for creating an exchange session now requires a redirect url parameter, which is a single URL that is already configured with Dwolla. ([Balance docs](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer), [Connect docs](https://developers.dwolla.com/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party))
```

Reply:
Yeah, that's what I was thinking